fastlane/Fastfile

```diff
@@ -330,7 +330,7 @@ platform :android do
 
     gradle(
       tasks: [
-        "androidSourcesJar", "androidJavadocJar", "publish --no-daemon --no-parallel"
+        "publish --no-daemon --no-parallel"
```

Comment:
Well, I guess publish should indeed build the jar file, and I guess it also adds the javadocs to the jar normally so as long as it's tested, I think this is ok.